template/core/data/src/main/java/io/bloco/core/data/repositories/BookRepository.kt

```diff
@@ -0,0 +1,19 @@
+package io.bloco.core.data.repositories
+
+import io.bloco.core.data.models.BookDetailsDto
+import io.bloco.core.data.models.BookDto
+import io.bloco.core.data.network.OpenLibraryService
+import javax.inject.Inject
+
+class BookRepository
+@Inject constructor(
+    private val openLibraryService: OpenLibraryService
+) {
+    suspend fun getBooks(): Result<List<BookDto>> =
```

Comment:
Here something we sometimes do is create a CollectionOfBooksDto. This allows for us to use pagination later one with lesser refactors. But I guess it might be overkill... not sure...